packages/mdc-slider/index.js

```diff
@@ -116,6 +119,28 @@ export class MDCSlider extends MDCComponent {
       setTrackStyleProperty: (propertyName, value) => {
         this.track_.style.setProperty(propertyName, value);
       },
+      setMarkerValue: (value) => {
+        this.pinValueMarker_.innerText = value;
+      },
+      appendTrackMarkers: (numMarkers) => {
+        const frag = document.createDocumentFragment();
+        for (let i = 0; i < numMarkers; i++) {
+          const marker = document.createElement('div');
+          marker.classList.add('mdc-slider__track-marker');
+          frag.appendChild(marker);
+        }
+        this.trackMarkerContainer_.appendChild(frag);
+      },
+      removeTrackMarkers: () => {
+        while (this.trackMarkerContainer_.firstChild) {
+          this.trackMarkerContainer_.removeChild(this.trackMarkerContainer_.firstChild);
+        }
+      },
+      setLastTrackMarkersStyleProperty: (propertyName, value) => {
```

Comment:
IMHO, the rule of thumb for mdc is to make the adapter as less ambiguous as possible, which it can almost read like a human sentence. The other components I feel have been taking this approach like `setStyleForTilesElement`, `setStyleForFlexibleRowElement`, `setStyleForTitleElement`. 

However, that being said, I totally can see value in this approach when we refactor to have global adapter. I think we should make changes that breaks our convention as a more holistic decision. WDYT?